docs/source/developingplugins.rst

```diff
@@ -8,6 +8,37 @@ to make linting Rules pluggable, which enable users to implement rules that
 are just too "organization specific" to be shared, or too platform specific
 to be included in the core library.
 
+.. note::
+
+    We recommend that the module in a plugin which defines all
+    of the hook implementations (anything using the ``@hookimpl`` decorator)
+    must be able to fully import before any rule implementations are imported.
+    More specifically, the SQLFluff must be able to both *import* **and**
```

Comment:
`the SQLFluff` doesn't read right

Reply:
good catch - I'll remove the `the`. It's not necessary.